dash_renderer/__init__.py

```diff
@@ -12,27 +12,17 @@
 from .version import __version__
 __file__
 
-_DEFAULT_REACT_VERSION = '15.4.2'
-_REACT_VERSION_TYPES = {'15.4.2', '16.2.0'}
+_DEFAULT_REACT_VERSION = '16.8.6'
+_REACT_VERSION_TYPES = {'16.8.6'}
 _REACT_VERSION_TO_URLS = {
-    '15.4.2': {
+    '16.8.6': {
```

Comment:
fwiw, no need to update the manifest as it wildcards react and react-dom versions